scripts/test-openssl.sh

```diff
@@ -280,18 +281,18 @@ printf "LD_LIBRARY_PATH: $LD_LIBRARY_PATH\n"
 # Set up wolfProvider
 cd ${WOLFPROV_DIR}
 if [ ! -e "${WOLFPROV_DIR}/configure" ]; then
-    ./autogen.sh &>> $LOG_FILE
-    ./configure --with-openssl=${OPENSSL_INSTALL_DIR} --with-wolfssl=${WOLFSSL_INSTALL_DIR} &>> $LOG_FILE
+    ./autogen.sh 2>&1 >> $LOG_FILE
+    ./configure --with-openssl=${OPENSSL_INSTALL_DIR} --with-wolfssl=${WOLFSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/test-openssl.sh line 284:
    ./autogen.sh 2>&1 >> $LOG_FILE
                 ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                         ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```
```
In scripts/test-openssl.sh line 285:
    ./configure --with-openssl=${OPENSSL_INSTALL_DIR} --with-wolfssl=${WOLFSSL_INSTALL_DIR} 2>&1 >> $LOG_FILE
                               ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                     ^--------------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                                                                            ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                                                                                    ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```